cephfs/directory.go

```diff
@@ -222,7 +217,7 @@ func (dir *Directory) List() (dirEntries, error) {
 }
 
 // Names returns a slice of only the name fields from dir entries.
-func (entries dirEntries) Names() []string {
+func Names(entries []*DirEntry) []string {
```

Comment:
If we're going to generalize the function we should probably make it generalize for both the readdir (DirEntry) and readdir_plus (DirEntryPlus) cases. We can probably do something like:
```go
type NamedEntry interface {
    Name() string
}

func Names(entries []NamedEntry) []string {
   // ... impl ...
}
```